src/views/conference/schedule/schedule.jsx

```diff
@@ -0,0 +1,122 @@
+                                <p>{item.Type}</p>
+                            </FlexRow>
+                        </div>
+                    </a>
+                );
+            } else {
+                return (
+                    <div key={item.rowid} className="agenda-item no-click">
+                        <h3>{item.Title}</h3>
+                        <FlexRow>
+                            <p>{item.Start} &ndash; {item.End}</p>
+                            <p>{item.Location}</p>
+                        </FlexRow>
+                    </div>
+                );
+            }
+        });
+    },
+    render: function () {
+        var tabClasses = {
+            'thursday': classNames({
+                'selected': (this.props.day === 'thursday')
+            }),
+            'friday': classNames({
+                'selected': (this.props.day === 'friday')
+            }),
+            'saturday': classNames({
+                'last': true,
+                'selected': (this.props.day === 'saturday')
+            })
+        };
+        return (
+            <div className="schedule">
+                <TitleBanner>
+                    <h1>
+                        Schedule
+                    </h1>
+                </TitleBanner>
+                <SubNavigation>
+                    <li className={tabClasses.thursday}
+                        onClick={this.handleScheduleChange.bind(this, 'thursday')}>
+                        <img src="/svgs/conference/expect/aug4-icon.svg" alt="August 4th Icon" />
+                        <span>Thursday</span>
+                    </li>
+                    <li className={tabClasses.friday}
+                        onClick={this.handleScheduleChange.bind(this, 'friday')}>
+                        <img src="/svgs/conference/expect/aug5-icon.svg" alt="August 5th Icon" />
+                        <span>Friday</span>
+                    </li>
+                    <li className={tabClasses.saturday}
+                        onClick={this.handleScheduleChange.bind(this, 'saturday')}>
+                        <img src="/svgs/conference/expect/aug6-icon.svg" alt="August 6th Icon" />
+                        <span>Saturday</span>
+                    </li>
+                </SubNavigation>
+                <div className="inner">
+                    {this.props.schedule.map(function (chunk) {
+                        return ([
+                            <h2 key={chunk.info.name} className="breaking-title">
+                                <span>{chunk.info.name} – {chunk.info.time}</span>
+                            </h2>,
+                            this.renderChunkItems(chunk.items)
+                        ]);
+                    }.bind(this))}
+                </div>
+            </div>
+        );
+    }
+});
+
+var mapStateToProps = function (state) {
+    return {
+        day: state.day,
+        schedule: state.schedule,
+        fetching: state.fetching,
```

Comment:
I think this was leftover from a previous version of the code. You never use `props.fetching` and `state.fetching` doesn't exist either.